spec/lib/convenient_service/support/dependency_container/entities/method_spec.rb

```diff
@@ -226,11 +252,26 @@
           end
         end
       end
+
+      describe "#to_kwargs" do
+        let(:kwargs) do
+          {
+            slug: slug,
+            scope: scope,
+            body: body,
+            alias_slug: alias_slug
+          }
+        end
+
+        it "returns true" do
```

Comment:
`it "returns kwargs representation"` (similar as in the other `to_kwargs`  specs)/